qemu/tests/cfg/numa_hmat.cfg

```diff
@@ -1,7 +1,7 @@
 - numa_hmat:
     type = numa_hmat
-    only x86_64
-    required_qemu = [4.2.0,)
+    only x86_64, aarch64
+    required_qemu = [7.2.0,)
```

Comment:
Refer L85：`required_qemu = (,7.2.0]`,  only run test `with_node_initiator` with qemu `7.2.0` is not a correct usage.